styleguide/tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "module": "es2015",
     "lib": ["es2018", "dom"],
     "outDir": "./dist",
-    "jsx": "preserve",
+    "jsx": "react",
```

Comment:
We should probably just preserve for now, not like it makes much of a difference since this is being overridden by the build process. Babel is currently handling JSX conversion AFAIK

Reply:
It won't build with just preserve, so seems like it isn't handled

Reply:
What error are you getting with `preserve`?

Reply:
https://app.circleci.com/jobs/github/smartcontractkit/chainlink/95476